cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -698,15 +699,22 @@ protected JsonObject asJson(KafkaConnectorSpec spec) {
         return updateStatusPromise.future();
     }
 
-    protected Future<ConfigMap> connectMetricsConfigMap(String namespace, KafkaConnectCluster connect) {
+    protected CompositeFuture connectMetricsAndLoggingConfigMap(String namespace, KafkaConnectCluster connect) {
```

Comment:
Using `CompositeFuture` in the return type is not very nice, because it means callers of this method need to know what the elements of the composite are, and CompositeFuture is not type safe. I think it would be much nicer to declare a class which holds the metrics cm and the logging cm, like this:

```java
public class MetricsAndLoggingCm {
    public ConfigMap metricsCm;
    public ConfigMap loggingCm;
    MetricsAndLoggingCm(ConfigMap metricsCm, ConfigMap loggingCm) {
        // ...
    }
}
```

and have this method return a `Future<MetricsAndLoggingCm>` (by doing `map` on the result of the `join`). That way it's all much more self-describing.